.github/workflows/pytest.yml

```diff
@@ -0,0 +1,38 @@
+name: Tests
+on: [pull_request]
+jobs:
+
+#  lint:
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v2
+#      - uses: actions/setup-python@v2
+#        with:
+#          python-version: '3.9'
+#      - name: Install requirements
+#        run: pip install flake8 pycodestyle
+#      - name: Check syntax
+#        run: |
+#          flake8 . --count --max-line-length=127 --show-source --statistics
```

Comment:
Can you move `--max-line-length=127` into a separate flake config file `.flake8` like in the docs: 
https://flake8.pycqa.org/en/latest/user/configuration.html